src/trading/cli/consts.ts

```diff
@@ -0,0 +1,165 @@
+import { PromptObject } from 'prompts'
+import { ALL_SUPPORTED_CHAIN_IDS, SupportedChainId } from '../../common'
+import { OrderKind } from '../../order-book'
+
+export const traderParametersSchema: PromptObject[] = [
+  {
+    type: 'select',
+    name: 'chainId',
+    message: 'Network',
+    choices: ALL_SUPPORTED_CHAIN_IDS.map((key) => ({
+      title: key.toString(),
+      value: SupportedChainId[key],
+    })),
+    initial: SupportedChainId.MAINNET,
+  },
+  {
+    type: 'password',
+    name: 'signer',
+    message: 'Signer`s private key',
+  },
+  {
+    type: 'text',
+    name: 'appCode',
+    message: 'Your app code (for analytics purposes)',
+  },
+]
+
+export const tradeBaseParametersSchema: PromptObject[] = [
+  {
+    type: 'select',
+    name: 'orderKind',
+    choices: [
+      { title: 'Sell', value: OrderKind.SELL },
+      { title: 'Buy', value: OrderKind.BUY },
+    ],
+    message: 'Choose operation for limit order RFQ: create, fill, cancel',
+  },
+  {
+    type: 'text',
+    name: 'sellToken',
+    message: 'Sell token address',
+  },
+  {
+    type: 'number',
+    name: 'sellTokenDecimals',
+    message: 'Sell token decimals',
+  },
+  {
+    type: 'text',
+    name: 'buyToken',
+    message: 'Buy token address',
+  },
+  {
+    type: 'number',
+    name: 'buyTokenDecimals',
+    message: 'Buy token decimals',
+  },
+  {
+    type: 'text',
+    name: 'amount',
+    message: 'Amount to trade (in units)',
+  },
+]
+
+export const tradeOptionalParametersSchema: PromptObject[] = [
+  {
+    type: 'select',
+    name: 'env',
+    choices: [
+      { title: 'Prod', value: 'prod' },
+      { title: 'Staging', value: 'staging' },
+    ],
+    message: 'Environment',
```

Comment:
Set default to `prod`

```suggestion
    message: 'Environment',
    initial: 'prod',
```